version.swift

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/swift
+
+import Foundation
+
+/// Tag management
+
+struct Tag: CustomStringConvertible {
+    let major: Int
+    let minor: Int
+    let patch: Int
+
+    var description: String {
+        return "\(major).\(minor).\(patch)"
+    }
+
+    func nextPatch() -> Tag {
+        return Tag(major: major, minor: minor, patch: patch + 1)
+    }
+
+    func nextMinor() -> Tag {
+        return Tag(major: major, minor: minor + 1, patch: 0)
+    }
+
+    func nextMajor() -> Tag {
+        return Tag(major: major + 1, minor: 0, patch: 0)
+    }
+}
+
+final class TagManager {
+    func loadCurrentTag() -> Tag? {
+        let regex = "^[0-9]*\\.[0-9]*\\.[0-9]"
+        let gitTag = execute("git tag | sort -r | grep '\(regex)' | head -1 2> /dev/null") ?? ""
```

Comment:
I don't think `sort -r` does numeric sorting. I.e. given this list of versions: `4.1 4.2 … 4.10 4.10.1` it would be sorted as:

```
4.2
4.10.1
4.10
4.1
```

Reply:
I think you're right

Reply:
Should be fixed now.